tinygrad/tensor.py

```diff
@@ -207,7 +207,7 @@ def kaiming_normal(*shape, a:float = 0.01, **kwargs) -> Tensor:
   def deepwalk(self):
     def _deepwalk(node, visited, nodes):
       visited.add(node)
-      if node._ctx:
+      if getattr(node, "_ctx", None):
```

Comment:
can't you just use `hasattr` here?

Reply:
`hasattr(node, "_ctx")` is true when `node._ctx = None`
so `getattr(node, "_ctx", None):` is basically short for `if hasattr(node, "_ctx") and node._ctx:`